app/utils.py

```diff
@@ -26,3 +28,13 @@ def get_template_instance(template, values):
     return {
         SMS_TYPE: SMSMessageTemplate, EMAIL_TYPE: PlainTextEmailTemplate
     }[template['template_type']](template, values)
+
+
+def get_midnight_for_date(date):
```

Comment:
can you rename this to `get_utc_midnight_for_date` (and the same with the function below) just so we're clear that it needs adjusting if you care about what day it is (pete ruled that it doesn't matter for this for now but a rename would be good 👍 )